src/main/kotlin/services/userService.kt

```diff
@@ -0,0 +1,87 @@
+package services
+
+import com.google.gson.GsonBuilder
+import models.UserModel
+import org.json.JSONObject
+import repo.UserRepo
+import java.util.*
+
+class Service {
+
+    fun handleGetAll(): MutableList<UserModel> {
+        return UserRepo().getAll()
+    }
+
+    fun handleGet(id: String?): UserModel?{
```

Comment:
getById method if id is not there then method's existence does not make any sense, make id non-nullable in resource layer itself